torch/_dynamo/config.py

```diff
@@ -98,7 +98,7 @@ def is_fbcode():
 allow_ignore_mark_dynamic = False
 
 # Set this to False to assume nn.Modules() contents are immutable (similar assumption as freezing)
-guard_nn_modules = False if is_fbcode() else True
+guard_nn_modules = not is_fbcode()
```

Comment:
Just doing a null test to see rocm tests tail. Will revert back the last commit.